README.md

```diff
@@ -1,48 +1,55 @@
-# openEO Project Steering Committee Guidelines
+# openEO Project Steering Committee
 
-Author: Edzer Pebesma, mostly copied from https://gdal.org/development/rfc/rfc1_pmc.html#rfc-1
+This document describes how the openEO Project Steering Committee determines membership, and makes decisions on openEO project issues. The openEO project is concerned with all repositories (software and specifications etc.) found under https://github.com/Open-EO.
```

Comment:
Clarified here that it's no just software, but all repositories (software and specs)